src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtils.java

```diff
@@ -76,13 +74,12 @@ public static GenotypesContext subsetAlleles(final GenotypesContext originalGs,
                     final int PLindex = MathUtils.maxElementIndex(newLikelihoods);
                     newLog10GQ = GenotypeLikelihoods.getGQLog10FromLikelihoods(PLindex, newLikelihoods);
                 }
-
             }
 
-            final boolean useNewLikelihoods = newLikelihoods != null && (depth != 0 || GATKVariantContextUtils.isInformative(newLikelihoods));
+            final boolean useNewLikelihoods = newLikelihoods != null && newLikelihoods.length > 1;
```

Comment:
This line was the solution to a big set of differences with gatk3 genotypegvcfs.  If we change it we will no longer match gatk3, which will make it more difficult to validate haplotype caller.  We may want to hold off on any change here until we're finished with that.

Reply:
Yeah, this was a hard-fought line of code that we rescued from GATK3 after much agonized searching. I really don't think we can change this now -- I'd move that this PR be closed.